### DISCUSSION TITLE
Automatic calculation between tables

### DISCUSSION THREAD

**Discussion by bmscmoreira** [Budibase/budibase#12077] [Help]
Wondering if this is possible. Imagine the following scenario:

1) Table "A" is an inventory and has records such as:

Object 1: 10
Object 2: 20
Object 3: 5

2) Table "B" is a movements table.

What I would like to do is:

- when a user adds a new row to Table "B" with a movement, it automaticaly updates Table A. For example, user adds to table B

Object 2: -10 (minus 10)

Then Table A would reduce the inventory of Object 2 by 10 units, resulting in:

Object 1: 10
Object 2: 10
Object 3: 5

Hope this makes sense. Thanks.

**Comment by ConorWebb96** [comment]
Hey @bmscmoreira,

This would likely require and [automations](https://docs.budibase.com/docs/automation-actions) to trigger whenever a record is updated/created in the movements table. If you have some form of relationship link between the movements table and the inventory table. (Some way to uniquely identify the record so that you can query for that specific record in the automation so it can be targeted and updated appropriately.) You should be able to update the rows accordingly. It may require slight JS intervention to figure out how exactly the calculations needs to be done but shouldn't be very difficult to achieve.

I hope this information helps!

**Reply by melohagan** [reply]
Formulas may also work in this case! https://docs.budibase.com/docs/formula
